### DISCUSSION TITLE
Combo com PoComboFilter - Comportamento inesperado

### DISCUSSION THREAD

**Discussion by wsteixeira** [po-ui/po-angular#1150] [Tire suas dúvidas] [answered]
Olá,

Eu observei que o combo quando usado com o "PoComboFilter", o valor fica "undefined" quando o "value" é zero! isto está correto? Segue abaixo um exemplo para simulação.

[https://stackblitz.com/edit/po-ui-cfj8lb?devtoolsheight=33&file=src/app/combo-filter.service.ts](https://stackblitz.com/edit/po-ui-cfj8lb?devtoolsheight=33&file=src/app/combo-filter.service.ts)

**Comment by alinelariguet** [comment, marked as answer]
Boa tarde @wsteixeira,
Obrigada por reportar, estou analisando aqui também, porém é possível utilizar o (p-change) em conjunto com a propriedade p-emit-object-value, dai o p-change vai receber como argumento  PoComboOption referente à opção selecionada.
Veja se isso resolve para você.
Att.
